lib/checks/color/color-contrast.json

```diff
@@ -4,7 +4,14 @@
 	"options": {
 		"noScroll": false,
 		"ignoreUnicode": true,
-		"ignoreLength": false
+		"ignoreLength": false,
+		"boldValue": 700,
+		"boldPt": 14,
+		"nonBoldPt": 18,
+		"minNormalContrastRatio": 0,
```

Comment:
This is a bit misleading; this isn't the minimal contrast ratio, 4.5:1 is the minimal.